models/Position.java

```diff
@@ -0,0 +1,141 @@
+import java.lang.Math;
+
+class Position {
+	/**
+	 * Absis
+	 */
+	private double x;
+	
+	/**
+	 * Ordinate
+	 */
+	private double y;
+	
+	/**
+	 * Constructor initiate the position with random coordinate
+	 */
+	public Position() {
+		this.random();
+	}
+
+	/**
+	 * Constructor, initiate the position with x and y
+	 * @param x is the absis
+	 * @param y is the coordinate
+	 */
+	public Position(double x, double y) {
+		this.x = x;
+		this.y = y;
+	}
+
+	/**
+	 * @return Return the absis of the positon
```

Comment:
`@return the absis of the position`
hilangin `Return` nya

Reply:
berlaku untuk semua @return ya

Reply:
semua file?

Reply:
oh aman aman